spark/processing/3.3/py3/docker/py39/Dockerfile.cpu

```diff
@@ -0,0 +1,132 @@
+FROM 137112412989.dkr.ecr.us-west-2.amazonaws.com/amazonlinux:2
```

Comment:
New file? I thought we would already have one dockerfile and would just need to update it. Why do we need a new docker file?

Reply:
Ah I see. So, for each major version upgrade we create new folder(missed the folder 3.3 earlier)

Reply:
yeah.. that's right!